src/window/index.ts

```diff
@@ -252,6 +252,21 @@ export async function openPathUpdate() {
   }));
 }
 
+export async function openLegacyIntegrations(): Promise<void> {
+  const window = openDialog(
+    'LegacyIntegrationNotification',
+    {
+      title:  'This is the title',
```

Comment:
I'd like to suggest a different title for this window, maybe something along the lines of

```suggestion
      title:  'Rancher Desktop - Legacy Integrations',
```

Reply:
Whoops... I remember writing that and thinking "don't forget to fix this!"